index.bs

```diff
@@ -325,7 +325,7 @@ should be treated by the user agent as the user intending to grant read access t
 for the returned files. As such, at the time the promise returned by one of the [=local file system handle factories=]
 resolves, [=permission state=] for a descriptor with {{FileSystemPermissionDescriptor/handle}} set to the returned handle,
 and {{FileSystemPermissionDescriptor/mode}} set to {{"read"}}
-should be {{PermissionState/granted}}.
+should be {{PermissionState/"granted"}}.
```

Comment:
This is wrong, but the other PR builds on this one and will fix, correct?

Reply:
correct. This is addressed in #418